resources.md

```diff
@@ -10,7 +10,7 @@ It's often said that the best way to learn to write programs is to write program
 
 ## Python-specific exercises
 
-- [exercism](https://exercism.io/): <q>Code practice and mentorship for everyone</q>, with over 100 [Python-specific exercises](https://exercism.io/tracks/python).
+- [exercism](https://exercism.io/): <q>Code practice and mentorship for everyone</q>, with over 100 [Python-specific exercises](https://exercism.io/tracks/python). Exercism is free and open-source, which is nice. It requires installation of their submission software, which is a slightly cumbersome process, but once installed the submission system is smoother than many online exercise portals. Problems are not always well-specified, and it may at times be difficult to understand exactly what is required.
```

Comment:
In the interest of keeping this short and inviting, what about something like this?

```suggestion
- [exercism](https://exercism.io/): Crowd-sourced <q>code practice and mentorship for everyone</q>, with over 100 [Python-specific exercises](https://exercism.io/tracks/python). Installing their command-line interface can be <q>a little complicated</q> , but after that the submission system is smoother than many online exercise portals.
```

If you feel strongly about "Problems are not always well-specified", can that be expressed more positively/succinctly? I'd also probably use "Exercises" instead of "Problems". And I'm curious: can you share an example or two of such exercises?

Reply:
One example of unclear spec: https://exercism.io/my/solutions/4f3ea117875143d5ad8f8cf2fa892642

The problem specification, in full, is "Output the lyrics to 'The Twelve Days of Christmas'.", followed by the lyrics of the last verse of that song. ("in full" meaning the unique content - there is a lot of boilerplate repeated on every problem)

If you look at the tests, it's possible to work out that what they want is a function that recites verses of the song from some `start_verse` to some `end_verse`, inclusive, but I would expect this information to be included in the problem spec.

Reply:
I'm not sure how to express this more positively or succinctly. Some of the problem descriptions I looked at were fine, others were confusing or incomplete. 
I do think it's worth raising this flag, because encountering an unexpectedly confusing problem is likely to be demoralizing for some beginners.